Source/WebKit/UIProcess/Launcher/ProcessLauncher.h

```diff
@@ -191,6 +195,10 @@ class ProcessLauncher : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<P
     bool m_isLaunching { true };
     ProcessID m_processID { 0 };
 
+#if ENABLE(BUBBLEWRAP_SANDBOX)
+    XDGDBusProxy* m_dbusProxy { nullptr };
```

Comment:
Hm, no this isn't OK. We don't hold ownership with a raw pointer, or use naked `new` and `delete`.

I would change this to hold an `XDGDBusProxy` directly without using any pointer. What I would do is move `ProcessLauncher::LaunchOptions` out of the `ProcessLauncher` class, and forward declare that in XDGDBusProxy.h. Then you can safely remove the `#include "ProcessLauncher.h"` in XDGDBusProxy.h and add `#include "XDGDBusProxy.h"` in ProcessLauncher.h.

Reply:
I've changed it in this way.